### PR TITLE
Installed pre-commit checker

### PR DESCRIPTION
**Pull request by danich1**
This is a small pre-commit hook I added to the repository. This will make future commits pre-formatted by black and make it easier for me to remain consistent on how I code.

### CODE REVIEW

File: .pre-commit-config.yaml [hunk 1]
```diff
@@ -0,0 +1,21 @@
+repos:
+
+- repo: local
+  hooks:
+  - id: jupytext_format
+    name: jupytext_format
+    entry: jupytext --from ipynb --pipe black --check flake8 --pre-commit
+    pass_filenames: false
+    language: python
```
Comment (ben-heil) [inline .pre-commit-config.yaml:9]: Oooh I didn't know that jupytext could also run linters on notebooks

**Review by ben-heil — APPROVED** [review]
This is cool, I need to do something like this for formatting my notebooks

File: .pre-commit-config.yaml (hunk 1 above)
Comment (danich1) [reply .pre-commit-config.yaml:9]: Yeah I found this gem, which makes it so much easier to remain consistent with my code.

Comment (cgreene) [reply .pre-commit-config.yaml:9]: This is amazing. This should get spammed all over our slack!

Comment (ajlee21) [reply .pre-commit-config.yaml:9]: cool!

File: install.sh [hunk 2]
```diff
@@ -0,0 +1,5 @@
+conda env create --file environment.yml
```
Comment (ajlee21) [inline install.sh:1]: Really nice idea to keep the installation simple for users. I should add this to my repos as well!

**Review by ajlee21 — APPROVED** [review]
Really nice additions to make the repo a lot cleaner! Thanks for sharing the great tips!

**Pull request merged**